source/style-guide/quickstart.txt

```diff
@@ -86,12 +86,12 @@ the following ones to help you boost the effectiveness of your writing.
      - :ref:`use-correct-punctuation`
 
        :ref:`punctuation`
-   * - **Use sentence-style capitalization for all titles and
+   * - **Use headline-style capitalization for all titles and
```

Comment:
Suggestion:


I think that it could be helpful a couple items:

- That headline-style is another name for "title case" wherever it might need to be introduced 
- Unfortunately, I think we use both Identify headline-style we follow. I think it is [AP Style](https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case) based on capitalization of conjunctions longer than 3 letters like [this example](https://www.mongodb.com/docs/manual/reference/operator/aggregation/top/#find-the-top-score-across-multiple-games) (not necessarily representative of every instance).

Reply:
Will definitely add that title case is another word for headline-style. 
Regarding your second point, I'm a little unsure what you mean -- so some titles use headline-style rules, and some use AP Style capitalization rules? Is this something to correct in the docs themselves (change all the titles to headline-style or AP style), or something to mention in the style guide?

Reply:
There are multiple variations of headline-style / title case such as "AP" and "Chicago Manual of Style". I was mentioning that we use the AP style. Therefore, I think we should mention that it is "AP title case" and/or "AP headline-style" to differentiate it from any other style variation.

Reply:
Got it, I'll clarify that it's AP style